installers/conda/osx/create_bundle.sh

```diff
@@ -0,0 +1,271 @@
+  find "$bundle_contents" -name '*.plist' -delete
+}
+
+# Add required resources into bundle
+#   $1 - Location of bundle contents directory
+#   $2 - Name of the bundle (part before .app)
+#   $3 - Path to icon to copy
+function add_resources() {
+  local bundle_contents=$1
+  local bundle_name=$2
+  local bundle_icon=$3
+  cp "$HERE"/BundleExecutable "$bundle_contents"/MacOS/"$bundle_name"
+  chmod +x "$bundle_contents"/MacOS/"$bundle_name"
+  cp "$HERE"/qt.conf "$bundle_contents"/Resources/bin/qt.conf
+  cp "$bundle_icon" "$bundle_contents"/Resources
+}
+
+# Add required resources into bundle
+#   $1 - Location of bundle contents directory
+#   $2 - Name of the bundle (part before .app)
+#   $3 - Path to icon (only the basename is used)
+#   $4 - The version of the bundle
+function create_plist() {
+  local bundle_contents=$1
+  local bundle_name=$2
+  local bundle_icon=$3
+  local version=$4
+  bundle_plist="$bundle_contents"/Info.plist
+  cp "$HERE"/Info.plist.base "$bundle_plist"
+  add_string_to_plist "$bundle_plist" CFBundleIdentifier org.mantidproject."$bundle_name"
+  add_string_to_plist "$bundle_plist" CFBundleExecutable "$bundle_name"
+  add_string_to_plist "$bundle_plist" CFBundleName "$bundle_name"
+  add_string_to_plist "$bundle_plist" CFBundleIconFile "$(basename $bundle_icon)"
+  add_string_to_plist "$bundle_plist" CFBundleVersion "$version"
+  add_string_to_plist "$bundle_plist" CFBundleLongVersionString "$version"
+}
+
+# Add a key string-value pair to a plist
+#  $1 - Full path to the plist
+#  $2 - Key name
+#  $3 - Key string value
+function add_string_to_plist() {
+  local filepath=$1
+  local key=$2
+  local string_value=$3
+  /usr/libexec/PlistBuddy -c "Add :$key string $string_value" "$filepath"
+}
+
+function create_disk_image() {
+  local version_name=$1
+  echo "Creating disk image '$version_name.dmg'"
+  # Add additional DragNDrop facilities
+  #   Add link to Applications
+  #   Pad out the image so that we can make changes went mounted
+  #   Store a background image
+  ln -s /Applications "$BUILD_DIR"/Applications
+  mkdir "$BUILD_DIR/.background"
+  cp "$HERE/dmg_background.png" "$BUILD_DIR/.background/background.png"
+  padding_filename=".dummy-padding-file"
+  mkfile 1m "$BUILD_DIR/$padding_filename"
+
+  # Set Finder background
+  hdiutil create -ov -srcfolder "$BUILD_DIR" -volname "$version_name" -fs HFS+ -format UDRW "$TEMP_IMAGE"
+  volume_name=$(hdiutil attach "$TEMP_IMAGE" | perl -n -e '/\/Volumes\/(.+)/ && print $1')
+  trap "cleanup $volume_name" RETURN
+  volume_path=/Volumes/"$volume_name"
+  rm "$volume_path"/"$padding_filename"
+  sed -e "s/@BUNDLE_NAME@/$bundle_name/" "$HERE/dmg_setup.scpt.in" > "$TEMP_OSA_SCPT"
+  osascript "$TEMP_OSA_SCPT" "$version_name"
+  ensure_volume_detached "$volume_name"
+
+  # Create final compressed dmg
+  test -f "$version_name".dmg && rm -f "$version_name".dmg
+  hdiutil convert "$TEMP_IMAGE" -format UDZO -imagekey zlib-level=9 -o "$version_name".dmg
```

Comment:
Is it worth switching to `UDBZ` for bzip-2 rather than zlib? 
It should compress down a bit more and all of our users should be OSX 10.4+ anyway

Reply:
That's a good call. To be honest I just followed what CPack do but Conda for Qt requires 10.9+ I think so that should be fine.